config/settings.yml

```diff
@@ -267,9 +267,13 @@ saml:
   # TODO: update `relays` in each <ENV>-settings.local.yml.j2 file & make these relevant for localhost use
   relays:
     vetsgov: http://localhost:3001/auth/login/callback
-    vagov: ~
+    vagov: http://local.vagov/auth/login/callback
```

Comment:
Decided to include this in `settings.yml` since this is never `nil` in `dev`, `staging`, or `prod` settings.local.yml file.